tests/moe/test_trtllm_gen_fused_moe.py

```
@@ -1847,6 +1848,127 @@ def test_fp8_block_scale_moe_fused_shared_experts_reject_ep():
+        # Routing replay buffers use the un-fused [num_tokens, top_k] layout.
+        with pytest.raises(ValueError, match="routing_replay_out is not supported"):
+            op(
+                **op_kwargs,
+                **common_kwargs,
+                routing_method_type=RoutingMethodType.DeepSeekV3.value,
+                routing_replay_out=torch.empty((1, 1), dtype=torch.int16),
+            )
+
+        # Only DeepSeekV3 routing implements the fused shared-expert slots.
+        for method in (RoutingMethodType.MiniMax2, RoutingMethodType.Renormalize):
+            with pytest.raises(ValueError, match="only supported .* DeepSeekV3"):
```

Ruff (0.16.0)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)